src/platforms/apple/common/usage.mdx

```diff
@@ -52,5 +52,8 @@ This integration will swizzle some methods to create breadcrumbs e.g.: for `UIAp
 
 ## Known Limitations
 
+- With 5.x.x, we added <PlatformLink to="/configuration/releases/">Release Health</PlatformLink>, which allows you to monitor your releases' health
```

Comment:
Since it was exactly on 5.0.0 what if we just say that?

```suggestion
- With 5.0.0, we added <PlatformLink to="/configuration/releases/">Release Health</PlatformLink>, which allows you to monitor your releases' health
```